docker-compose.yaml

```diff
@@ -0,0 +1,56 @@
+version: "3.7"
+services:
+  api:
+    image: ${API_IMAGE:-gcr.io/athenian-1/api:latest}
+    restart: unless-stopped
+    entrypoint: ['/bin/sh']
+    command: >
+      -c "python3 -m athenian.api.models.state \
+            postgresql://${POSTGRES_USER:-api}:${POSTGRES_PASSWORD:-api}@postgres:5432/state &&
+          python3 -m athenian.api.invite_admin \
```

Comment:
Why do we make another admin invitation if we are using the staging dumps?

Reply:
true, I was initially willing to dump only `metadata`, but yup better to dump everything and remove this 👍